src/main/java/Daniela_Chihai/com/Endava/SoA5/Java_Collections/User.java

```diff
@@ -0,0 +1,39 @@
+package Daniela_Chihai.com.Endava.SoA5.Java_Collections;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class User {
+    private long id;
+    private String username;
+    private Map<String, Address> addresses;
+
+    public User(long id, String username, Map<String, Address> addresses) {
+        this.id = id;
+        this.username = username;
```

Comment:
Please use the approach as into Items Class, and initialize the Map into default constructor.